services/datamanager/builtin/file_deletion.go

```diff
@@ -130,6 +130,7 @@ func deleteFiles(ctx context.Context, syncer datasync.Manager, deleteEveryNth in
 					}
 					return err
 				}
+				logger.Infof("successfully deleted %s", d.Name())
```

Comment:
Do we know how many logs per second this will add to viam server?

Reply:
I don't think there's a great way _to_ know about this. If Disk < 10%, we will have 1 line out of every 5 files until we cross that threshold. But, without having every deleted file logged, it's very challenging to see if the deleted files are what could be causing our issues. 

We can always remove the log (or make it a DEBUG) if this gets too unwieldy. But just for the time being, would be extremely helpful

Reply:
+1 for `DEBUG` level. we can also collect all deleted filenames and output them as 1 log line

Reply:
Do you know if `DEBUG` is synced? I was only worried because I didn't see any `DEBUG` logs in the console though I'm seeing some in Mongo on the fleet-kitchen-sink org

Reply:
Thanks y'all. I synced with @etai-shuchatowitz. I think what is here is fine. 

Debug level logging isn't going to work in the short term as we have customers which need this info which are not running at debug level.

If we were to be deleing a lot of files and try to append all the filenames into a single log, that might go over the data truncation limit.

I'm comfortable with this change as is.